index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desafio dos diamantes</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <select name="" id="select_diamante">
+        <option value="" selected>Selecione uma opção</option>
+    </select>
+
+    <div class="mostra_diamante" id="mostra_diamante"></div>
+    <script src="index2.js"></script>
```

Comment:
```suggestion
    <script src="index.js"></script>
```

Não faz sentido colocar um número no arquivo do javascript, qual o motivo principal do 2 ? apenas distinguir de um antigo index1?

Reply:
alterado